src/main/java/core/basesyntax/MainApp.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class MainApp {
+    public static final int FIGURES_AMOUNT = 6;
+
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[FIGURES_AMOUNT];
+
+        for (int i = 0; i < figures.length; i++) {
+            if (i < FIGURES_AMOUNT / 2) {
+                figures[i] = figureSupplier.getRandomFigure();
+            } else {
+                figures[i] = figureSupplier.getDefaultFigure();
+            }
+        }
+        for (Figure figure : figures) {
+            figure.getArea();
```

Comment:
Why did you make this call and then ignore result of it?

Reply:
get.Area is already implemented in a print method so it has no sense to call it 2 times, don't know why I did it)